Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :tag => "v0.7.2"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hadoop", :because => "both install `yarn` binaries"
+  conflicts_with "node", :because => "both install `node` binaries"
+  conflicts_with "npm", :because => "both install `npm` binaries"
+  conflicts_with "yarn", :because => "both install `yarn` binaries"
+
+  def install
+    ENV["VOLTA_HOME"] = prefix.to_s
+    system "./dev/unix/volta-install.sh", "--release", "--skip-setup"
```

Comment:
Does this download any dependencies?

Reply:
No, everything is compiled locally. I made sure I was able to install this without being connected.